src/legacy/core_plugins/kibana/index.js

```diff
@@ -54,6 +54,8 @@ export default function (kibana) {
         defaultAppId: Joi.string().default('home'),
         index: Joi.string().default('.kibana'),
         disableWelcomeScreen: Joi.boolean().default(false),
+        autocompleteTerminateAfter: Joi.number().default(10000),
```

Comment:
Looks like you missing an order of magnitude from the existing value. Can you make this `100000` instead of `10000`